packages/pages/src/dev/server/ssr/generateTestData.ts

```diff
@@ -151,3 +142,31 @@ export const generateTestDataForPage = async (
     });
   });
 };
+
+const addCommonArgs = (
+  featuresConfig: FeaturesConfig,
+  featureName: string,
+  locale: string
+) => {
+  // We need to specially handle the JSON when running on windows due to an existing bug/behavior in Powershell where inner quotes are
+  // stripped from strings when passed to a third-party program. Read more: https://stackoverflow.com/questions/52822984/powershell-best-way-to-escape-double-quotes-in-string-passed-to-an-external-pro.
```

Comment:
Is this equally a problem in the _output_? We run JSON.parse on the output of this command.

Reply:
No.